woocommerce/wc-admin
# Display table placeholder instead of empty table when loading search terms

Fixes #1676.

Prevent the report table showing an empty state when loading the search terms, instead, display the table placeholder.

### Screenshots
_Before (notice an empty table is visible for a fraction of a second):_
![empty-table](https://user-images.githubusercontent.com/3616980/53256054-4d9c6e00-36c7-11e9-9254-4ceb65570bc3.gif)

_After:_
![search-3](https://user-images.githubusercontent.com/3616980/53288521-59506900-3789-11e9-8905-41646a04309b.gif)

### Detailed test instructions:
- Go to the _Products_ report, for example.
- Search for a product using the table search input.
- Verify the table is never empty: it goes from displaying all products → placeholder → displaying the searched product.
- Repeat the same for the other reports that have search inputs in the tables.
